src/Components/Items.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { BsTrash3Fill } from "react-icons/bs";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+export default class Items extends React.Component {
+  render() {
+    const items = this.props.itemsList.map((item, key) => {
```

Comment:
If I compare `ItemsList` with `Items`, I don't see how the list is an actual list. You are rendering the list in this component. To correct this, I would run this map in the `ItemsList` component and pass down the updater etc. functions to the `Item` component, within the map, which will then determine how the updating/deletion etc. works

Reply:
in ItemsList it should look like this:

```
render() {
    return {this.itemsList.map((item, key) => {
      return <Item item={item} key={key} />
}}
}
```